roles/kustomize_deploy/tasks/generate_base64_patches_from_tree.yml

```diff
@@ -8,14 +8,27 @@
         paths: "{{ cifmw_architecture_user_kustomize_base_dir }}"
         patterns: "*.yml,*.yaml"
         recurse: true
-      register: _cifmw_kustomize_deploy_yaml_patches_find
+        file_type: "{{ item }}"
+      register: _kustomize_deploy_yaml_files
+      loop:
+        - file
+        - link
       delegate_to: localhost
 
+    - name: Set list of config files in configs directory.
+      ansible.builtin.set_fact:
+        _cifmw_kustomize_deploy_yaml_patches_find: >-
+          {{
+            item.files | default([]) +
+            _cifmw_kustomize_deploy_yaml_patches_find | default([])
+          }}
+      loop: "{{ _kustomize_deploy_yaml_files.results | default([]) }}"
+
     - name: Fetch the content of the kustomize patches
       ansible.builtin.slurp:
         path: "{{ item.path }}"
       register: _cifmw_kustomize_deploy_yaml_patches_slurp
-      loop: "{{ _cifmw_kustomize_deploy_yaml_patches_find.files | default([]) }}"
+      loop: "{{ _cifmw_kustomize_deploy_yaml_patches_find | default([]) }}"
```

Comment:
I think you can get rid of the new task, the one before this one and make this one like this:
```yaml
    - name: Fetch the content of the kustomize patches
      ansible.builtin.debug:
        msg: "{{ item.path }}"
      register: _cifmw_kustomize_deploy_yaml_patches_slurp
      loop: >-
        {{
          _kustomize_deploy_yaml_files.results |
          map(attribute='files') | flatten
        }}

```

Reply:
I just tested it and it looks much better. Thanks, @pablintino